### PR TITLE
MD-2296: Changes to schema change due to empty batch test cases

### PR DESCRIPTION
**Pull request by prasadns14**
This PR has the following changes -
1) Modified the empty batch test cases to use 1 minor fragment per region/table
2) Marked one test case as fail (DRILL-5822)

@cchang738, please review

### CODE REVIEW

File: framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseScan1.fail [hunk 1]
```diff
@@ -1 +1,3 @@
+alter session set `planner.slice_target`=1;
 select * from schema_change_empty_batch_part t order by t.data.p_partkey limit 5;
+alter session set `planner.slice_target`=100000;
```
Comment (cchang738) [inline framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseScan1.fail]: Why this file still with .fail extension? The associated bug is fixed, right? And now the test should pass? Also use set/reset.

File: framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbase_5480_1.sql [hunk 2]
```diff
@@ -1,3 +1,5 @@
-alter session set `planner.enable_broadcast_join`=false;
+set `planner.enable_broadcast_join`=false;
+set `planner.slice_target`=1;
 select * from hbase.customer1 c, cp.`tpch/orders.parquet` o where cast(c.orders.id as bigint) = o.o_orderkey and c.orders.id <= '200';
-alter session set `planner.enable_broadcast_join`=true; 
+reset `planner.slice_target`;
+set `planner.enable_broadcast_join`=true;
```
Comment (cchang738) [inline framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbase_5480_1.sql]: Can we confirm before run the planner.enable_broadcast_join was indeed set to true?

File: framework/resources/Functional/schema_change_empty_batch/json/emptyJsonScan1.fail [hunk 3]
```diff
@@ -0,0 +1,3 @@
+alter session set `planner.slice_target`=1;
+select * from `json/part_small` order by p_partkey;
+alter session set `planner.slice_target`=100000;
```
Comment (cchang738) [inline framework/resources/Functional/schema_change_empty_batch/json/emptyJsonScan1.fail]: Same here.

File: framework/resources/Functional/schema_change_empty_batch/maprdb/binary_hbase/emptyMaprDBScan1.fail [hunk 4]
```diff
@@ -1 +1,3 @@
+alter session set `planner.slice_target`=1;
 select * from `/drill/testdata/schema_change_empty_batch/maprdb/binary/part` t order by t.data.p_partkey limit 5;
+alter session set `planner.slice_target`=100000;
```
Comment (cchang738) [inline framework/resources/Functional/schema_change_empty_batch/maprdb/binary_hbase/emptyMaprDBScan1.fail]: And here.

File: framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBScan1.fail [hunk 5]
```diff
@@ -1 +1,3 @@
+alter session set `planner.slice_target`=1;
 select * from `maprdb/binary/part` t order by t.data.p_partkey limit 5;
+alter session set `planner.slice_target`=100000;
```
Comment (cchang738) [inline framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBScan1.fail]: And here.

File: framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBScan4.sql [hunk 6]
```diff
@@ -0,0 +1,3 @@
+alter session set `planner.slice_target`=1;
+select * from `maprdb/binary/empty`;
+alter session set `planner.slice_target`=100000;
```
Comment (cchang738) [inline framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBScan4.sql]: use reset.

File: framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseScan1.fail (hunk 1 above)
Comment (prasadns14) [reply framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseScan1.fail]: Addressed comments.

There were two JIRS's associated with this test case - DRILL-5743, DRILL-5706. Only DRILL-5706 is fixed

File: framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbase_5480_1.sql (hunk 2 above)
Comment (prasadns14) [reply framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbase_5480_1.sql]: Addressed comments. 
Using reset for planner.enable_broadcast_join

File: framework/resources/Functional/schema_change_empty_batch/json/emptyJsonScan1.fail (hunk 3 above)
Comment (prasadns14) [reply framework/resources/Functional/schema_change_empty_batch/json/emptyJsonScan1.fail]: Addressed

File: framework/resources/Functional/schema_change_empty_batch/maprdb/binary_hbase/emptyMaprDBScan1.fail (hunk 4 above)
Comment (prasadns14) [reply framework/resources/Functional/schema_change_empty_batch/maprdb/binary_hbase/emptyMaprDBScan1.fail]: Addressed

File: framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBScan1.fail (hunk 5 above)
Comment (prasadns14) [reply framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBScan1.fail]: Addressed

File: framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBScan4.sql (hunk 6 above)
Comment (prasadns14) [reply framework/resources/Functional/schema_change_empty_batch/maprdb/binary_maprdb/emptyMaprDBScan4.sql]: Addressed

File: framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseScan1.fail (hunk 1 above)
Comment (cchang738) [reply framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbaseScan1.fail]: Can you change to reset? Setting slice_target to 100,000 doesn't make much sense.

File: framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbase_5480_1.sql (hunk 2 above)
Comment (cchang738) [reply framework/resources/Functional/schema_change_empty_batch/hbase/emptyHbase_5480_1.sql]: Did you actually use reset?

**Comment by cchang738** [comment]
Can you rerun pre-commit test? If no issue found then merge (+1).

**Comment by prasadns14** [comment]
Enabled all the empty batch test cases. @cchang738 please review

**Comment by cchang738** [comment]
+1

**Pull request merged**